### PR TITLE
Backpack Nerf

### PR DESCRIPTION
**Pull request by LordFowl**
Backpacks can only be opened in your hands. Satchels can be opened anywhere, but have reduced storage capacity.

### CODE REVIEW

**Comment by LordFowl** [comment]
!review

**Comment by ParadoxSpace** [comment]
What about duffel bags?

**Comment by LordFowl** [comment]
They are unaffected.

File: code/game/objects/items/weapons/storage/storage.dm [hunk 1]
```diff
@@ -36,6 +36,7 @@
 	var/allow_quick_gather	//Set this variable to allow the object to have the 'toggle mode' verb, which quickly collects all items from a tile.
 	var/collection_mode = 1  //0 = pick one at a time, 1 = pick all on tile
 	var/use_sound = "rustle"	//sound played when used. null for no sound.
+	var/openned_in_hands = 0 //can only be openned if held in hands
```
Comment (ParadoxSpace) [inline code/game/objects/items/weapons/storage/storage.dm]: opened

**Comment by AIMMO2001** [comment]
I think this would unnecessarily make things way more complicated. 

The two arguments I heard were it being used for better RP and to prevent people just walking around with guns in their bags and pulling them out in the matter of one click to shoot you.

For the first:
It's my own opinion only here but, I think we shouldn't sacrifice fun playing the game for such minor additions to realism. In the end, it changes from two short clicks to three short clicks.

For the second:
I doubt it will have the wished effect. Smaller weapons should still work with the satchel just as well, the backpack wouldn't reduce your time to pull too much. (one more click). And as its something you would need to do on the regular people that don't expect it will be just as off guard as normally.
I think this issue can be better fought with retracted versions of most larger weapons, which makes the weapon either retracted ((short enough to fit into backpacks)) or ready ((Too long for backpacks but can well .. shoot)) with a progress bar for changing between those modes.

If these changes are done (partially) for other reasons my view might change, but I simply think its a hurdle to playing the game more than anything else.

**Comment by veraknocks** [comment]
highly unnecessarily overcomplicated as aimmo says. may the forces of Good triumph over this force of evil.

**Comment by skull132** [comment]
On the flip side.

You can use backpack storage to cheese, _a lot._ Basically, if you're smart, you will always have the backpack menu open ingame. Basically giving you instant access to all of the items you have there, resulting in no benefits being provided from the usage of other, more limited storage. And nerfing backpacks in some capacity has been discussed before.

Here's my counter proposal, which isn't necessarily as obstructive, but would still serve to balance them:
* Make the backpack's storage window close when you move, provided the backpack is on your back.
* Make opening said storage window take a second or two or three if the backpack is on your back.

This would still require you to pause to look at your backpack, it would remove the element of cheese I described earlier, but it would not be _as_ outright gameplay flow breaking as having to take the thing into your hands.

You could also add a third note where taking large objects (`w_class` > 2) take a second or two of time.

**Comment by OneOneThreeEight** [comment]
Only way I'd approve this is if satchels were left as-is but backpack storage got buffed. Accomplishing the same but different result, so to speak.

**Comment by LordFowl** [comment]
Both of Skull's solutions, in particular the second one, are inferior to the one presented here.

**Pull request closed without merging**